core/queryalgebra/model/src/main/java/org/eclipse/rdf4j/query/algebra/AbstractQueryModelNode.java

```diff
@@ -170,16 +170,16 @@ public void setTotalTimeNanosActual(long totalTimeNanosActual) {
 	/**
 	 * @return Human readable number. Eg. 12.1M for 1212213.4 and UNKNOWN for -1.
 	 */
-	static String toHumanReadbleNumber(double number) {
+	static String toHumanReadableNumber(double number) {
```

Comment:
Thanks for catching this typo. Should we extract this into an utility at somepoint?